sdk/cognitiveservices/azure-cognitiveservices-knowledge-qnamaker/azure/cognitiveservices/knowledge/qnamaker/authoring/operations/knowledgebase_operations.py

```diff
@@ -407,7 +407,7 @@ def create(
     create.metadata = {'url': '/knowledgebases/create'}
 
     def download(
-            self, kb_id, environment, custom_headers=None, raw=False, **operation_config):
+            self, kb_id, environment, source=None, changed_since=None, custom_headers=None, raw=False, **operation_config):
```

Comment:
this seems like a breaking change

Reply:
it's borderline, customer_headers is a generated header from autorest (not the service), the likeliness that someone is using positional arguments all the way to custom_headers is very low, and we never increased a major version just for it (and never got any issue opened)